Example/Tests/CSRTests.swift

```diff
@@ -59,6 +73,13 @@ final class CSRTests: XCTestCase {
   private var expectedCSR: String {
     return TestData.opensslCSR.stringTrimmingWhitespacesAndNewlines
   }
+  
+  private var expectedCSRWithUmlauts: String {
+    /* Generated with
```

Comment:
Just a note that it might better to put this kind of things as documentation:
```suggestion
/// Generated with openssl req -new -utf8 -key ./Example/Tests/Files/OpenSSL/openssl-private-key-pkcs1-pem -out ./Example/Tests/Files/CSR/openssl-csr-with-umlauts -subj "/C=DE/ST=Baden-Württemberg/L=Nürnberg/O=Vüvy/OU=ÄIT/emailAddress=test@vivy.com/UID=someÜID/GN=Gǖvenname/SN=Söörnamê"
private var expectedCSRWithUmlauts: String {
```